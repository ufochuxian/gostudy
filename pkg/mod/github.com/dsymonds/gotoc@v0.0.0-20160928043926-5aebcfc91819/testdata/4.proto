// Group.

syntax = "proto2";

message A {
  optional group B = 1 {
    required string x = 2;
  }
}
