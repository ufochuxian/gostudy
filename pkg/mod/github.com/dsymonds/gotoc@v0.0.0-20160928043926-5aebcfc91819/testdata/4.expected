file: <
  name: "4.proto"
  message_type: <
    name: "A"
    field: <
      name: "b"
      number: 1
      label: LABEL_OPTIONAL
      type: TYPE_GROUP
      type_name: ".A.B"
    >
    nested_type: <
      name: "B"
      field {
        name: "x"
        number: 2
        label: LABEL_REQUIRED
        type: TYPE_STRING
      }
    >
  >
>
